.dockerignore

```diff
@@ -1,3 +1,3 @@
-lib
+lib/*
```

Comment:
Yes.
There is a make target like make enclave that assumes the lib directory exists.
However, when adding the lcp directory to the image in docker build, an error occurs because the lib directory is not included.